src/palette/qml/MuseScore/Palette/internal/PaletteTree.qml

```diff
@@ -403,7 +403,7 @@ StyledListView {
                 paletteTree.implicitWidth = Math.max(paletteTree.implicitWidth, w);
             }
 
-            property bool expanded: filter.length || model.expanded
+            property bool expanded: searchOpened || model.expanded
```

Comment:
Nitpick: **paletteTree**.searchOpened || model.expanded (better for performance). Also, looks like we can make this property readonly

Reply:
Didn't know **paletteTree.searchOpened** would be faster. Will update it. Thanks!

Reply:
It's faster because otherwise QML will have to figure out at runtime which object the `searchOpened` property belongs to. I believe Qt were actually planning to remove support for "unqualified lookups" in version 6, but this has not happened. Nevertheless, they recommend against it for performance reasons: https://doc.qt.io/qt-6/qml-codingconventions.html#unqualified-access

Reply:
Thanks @cbjeukendrup, this makes perfect sense.